scipy/optimize/_linprog.py

```diff
@@ -582,7 +582,7 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
     ...               options={'maxiter': 4})
     >>> print(res)
         con: array([], dtype=float64)
-        fun: -21.35207150630407
+        fun: -21.35207150630407 # may vary
```

Comment:
Alternatively, can just do `>>> res.fun` instead. Assuming the rest of the stuff is not relevant, that is. (Or print those attributes which are relevant.

Reply:
I'm not sure that helps, `res.fun` actually changes I believe because the algorithm isn't 100% stable. This probably came in when we dropped Python 3.7 (?), and bumping the job to 3.8 changes the exact result. So it's either `# may vary` or just changing the result to the new value.

Reply:
How much does it change--- Numerical noise or multiple minima?

Reply:
In the example at the top, there is now a 5 tacked on to the end of the output. Has someone seen a more substantial difference?

Reply:
every time I see the error reported it is displayed as a difference between the `5` appended vs. not

Reply:
I suppose Evgeni is suggesting that this will fall within the threshold of floating-point equivalence instead of triggering an error on a printed string comparison?

Reply:
okay this failure is getting annoying, and the fix is okay - I'll just merge this.